package.json

```diff
@@ -71,11 +71,11 @@
     "lint-staged": "^8.1.4",
     "prettier": "^1.16.4",
     "shx": "^0.3.2",
-    "typescript": "~3.4",
+    "typescript": "~3.5",
     "web-ext": "^3.0.0"
   },
   "dependencies": {
-    "react-scripts": "3.0.0"
+    "react-scripts": "3.0.1"
```

Comment:
Can you try to downgrade to `3.0.0` and execute `git clean -xdf && yarn install && yarn build`